cmd/main.go

```diff
@@ -217,12 +229,13 @@ func runServer(ctx *cli.Context) error {
 	port := ctx.Uint64("port")
 	sleep := ctx.Uint64("sleep")
 	numOpersLoop := ctx.Uint64("opers")
+	writeTout := ctx.Uint64("writetout")
```

Comment:
wTimeout or writeTimeout looks better, IMHO

Reply:
I agree, I think writeTimeout is a good option as writetout you are only "saving" 3 chars :-)

Reply:
This is just a DS CLI testing tool.